Solutions/ApacheHTTPServer/Analytic Rules/ApacheMultipleServerErrorsRequestsFromSingleIP.yaml

```diff
@@ -0,0 +1,28 @@
+id: 1bf246a2-3af9-11ec-8d3d-0242ac130003
+name: Apache - Multiple server errors from single IP
+description: |
+  'Detects multiple server errors from one source in short timeframe'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheHTTPServer
+    dataTypes:
+      - ApacheHTTPServer
+tactics:
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T1498
+  - T1190
+  - T1133
+query: |
+  let threashold = 100;

```

Comment:
Probably a small typo but if wanted we can change the spelling to 'threshold' here and in the query.